src/ace_time/hw/stmrtc.h

```diff
@@ -0,0 +1,50 @@
+/*
+ * MIT License
+ * Copyright (c) 2018 Brian T. Park, Anatoli Arkhipenko
+ */
+
+#ifndef ACE_TIME_HW_STMRTC_H
+#define ACE_TIME_HW_STMRTC_H
+
+#if ! defined(UNIX_HOST_DUINO)
+#if defined(ARDUINO_ARCH_STM32)
+
+#include <stdint.h>
+#include <STM32RTC.h>
+
+namespace ace_time {
+namespace hw {
+
+class HardwareDateTime;
+
+/**
+ * A class that reads and writes HardwareDateTime and HardwareTemperature from a
+ * STM RTC chip. This class is designed to access just enough features of the
+ * RTC chip to implement the ace_time::StmRTCClock class. It is not
+ * meant to provide access to all the features of the RTC chip.
+ 
+ * Requires https://github.com/stm32duino/STM32RTC
+ */
+class STMRTC {
+  public:
+    /** Constructor. */
+    explicit STMRTC();
+
+    /** Read the time into the HardwareDateTime object. */
+    void readDateTime(HardwareDateTime* dateTime) const;
+
+    /** Set the STM with the HardwareDateTime values. */
+    void setDateTime(const HardwareDateTime& dateTime) const;
+
+    bool isTimeSet() const;
+  private:
+    STM32RTC* rtc;
```

Comment:
nit: consistency: Can you change this to either `mRtc`, or these days, my new code uses `rtc_` with a trailing underscore. (Don't want to use a prefix underscore since the C standard reserves all symbols starting with underscore and Capital (i.e. `_[A-Z].*`) and it's too easy to accidentally slide into that)